src/extension/sldService/src/test/java/org/geoserver/sldservice/rest/ColorRampTest.java

```diff
@@ -16,7 +16,7 @@
 import org.geoserver.sldservice.utils.classifier.impl.RedColorRamp;
 import org.junit.Test;
 
-public class ColorRampTest extends SLDServiceBaseTest {
+public class ColorRampTest {
```

Comment:
This test is a true unit one, does not need to be a GeoServerSystemTest subclass